rita/src/exit.rs

```diff
@@ -175,6 +175,7 @@ fn main() {
             .resource("/setup", |r| r.method(Method::POST).with2(setup_request))
             .resource("/list", |r| r.method(Method::POST).with(list_clients))
             .resource("/exit_info", |r| r.method(Method::GET).with(get_exit_info))
+            .resource("/rtt", |r| r.method(Method::GET).with(rtt))
```

Comment:
this needs to be api doc'd, make a folder doc in the top level and add a markdown format file called endpoints.md with api documentation.